tests/network/conftest.py

```diff
@@ -190,12 +190,6 @@ def brcnv_vma_with_vlan_1(
     )
 
 
-@pytest.fixture(scope="session")
-def skip_if_not_ovn_cluster(ovn_kubernetes_cluster):
```

Comment:
WDYT about adding this to a "requirements" fixture and asserting on it to fail quickly.
It is basically expressing that this is what we expect.

Reply:
This would be unnecessary. We don't support SDN since 4.14. No upgrade path exists from 4.14->4.18 (that we actually test, legit path would be from 4.17 and 4.16 - both does not support fresh install SDN). So there is no point checking it.

Reply:
There is a hidden assumption that CNV is not supporting other network providers.
But if you are saying that there is no chance we will run this on a cluster which has no OVN-K, fine by me.

Reply:
I got the same feedback from other sources. We assume OVN-K is a must and covering other providers is the partner responsibility.

So lets resolve this thread, thanks.